src/common/module/logger.ts

```
@@ -0,0 +1,18 @@
+import * as path from 'path';
+import * as fs from 'fs'
+
+export function logErrorToFile(error: any, logFileName = 'error.log') {
+    const logDir = path.join(process.cwd(), 'logs')
```

ESLint

[error] Replace `··const·logDir·=·path.join(process.cwd(),·'logs')` with `const·logDir·=·path.join(process.cwd(),·'logs');`

(prettier/prettier)